Formula/h/harlequin.rb

```diff
@@ -0,0 +1,178 @@
+class Harlequin < Formula
+  include Language::Python::Virtualenv
+
+  desc "Easy, fast, and beautiful database client for the terminal"
+  homepage "https://harlequin.sh"
+  url "https://files.pythonhosted.org/packages/bb/c1/0fc47822534938e73f070eb485059be414bb6ee274ac05e1d5d695fb4c24/harlequin-1.25.2.tar.gz"
+  sha256 "8319da69b07afd063e1296ebba1569374daa171489b97bd35d2dbdae802e6144"
+  license "MIT"
+
+  depends_on "apache-arrow" => :build
```

Comment:
this should be also runtime dep

```suggestion
  depends_on "apache-arrow"
```